### ISSUE TITLE
Plot window dimensions, and visibility

### ISSUE THREAD

**Issue by erigler** [usgs/geomag-plots#49]
It should be possible to adjust  plot window dimensions, or at least their height. This is NOT the same as setting the data range, and is primarily to give the user some control over how many plots appear on a single page. Currently, the plot window heights are unpredictably re-sized if/when the browser window is resized...at least this is what happens on my Macbook.

Ideally this would be a per-plot-window option, so users could effectively collapse unwanted windows, and create a custom view of select observatories, or select channels.

**Comment by jmfee-usgs** [comment]
The changing you see are the desktop/mobile responsive layout changes:
https://github.com/usgs/geomag-plots/blob/master/src/htdocs/js/TimeseriesCollectionView.js#L68

Basically, on smaller screens the smaller plots were considered less usable, we can discuss how to improve.

**Comment by erigler** [comment]
I'm not sure this is what I'm referring to @jmfee-usgs . What I see is:

1) load geomag.usgs.gov/plots in a normally sized browser window;
2) grab the right edge of the browser window and change its width;
3) the plots scale (h & w) with the changing width;

Maybe this should be another bug report, but I'd much prefer the window heights to NOT change if I change the width of my browser window. I think the fact that the plot window *widths* change is a good thing, but I can imagine allowing this while also preventing the heights from changing might be difficult programmatically.

**Comment by jmfee-usgs** [comment]
You're correct, that is related but different than what I originally mentioned.  The plot aspect ratio is the related portion, and plots resize to fill the available area.  We can discuss how to improve.

Currently we support a "mobile" layout, where plots are larger to be easier to interact with; and a "desktop" layout, where more plots are available for comparison.  Any suggestions are welcome.

**Comment by cafinn-usgs** [comment]
A related issue, that was mentioned early on, is there's no way to force the display to show all plots in one window, without the need to scroll.  This is useful for visual comparisons between observatory traces, and having to scroll up and down and only see five traces at a time is not very useful.

**Comment by emcwhirter-usgs** [comment]
This is a workaround for now (not a fix): Making the browser width smaller also makes all plots shorter so that more of them fit on the screen at the same time. It is not ideal because you also see less detail in each plot, but I think it could be helpful.

**Comment by hasimpson-usgs** [comment]
TimeseriesView should offer a compact and expanded mode. Manage this using css where possible.
